notebooks/Emukit-constrained-optimization.ipynb

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
<p>This this way of applying constraints allows violations, would be great to have at least a note of what people should do if that happens.</p>

 _Reply via <a href='https://app.reviewnb.com/amzn/emukit/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Added in the intro and pointed out this only works with positive acquisition functions</p>